react/Wrapper.tsx

```diff
@@ -99,6 +101,8 @@ const Wrapper = withToast(function Wrapper(props: Props) {
   const areAllSkuVariationsSelected =
     !isEmptyContext && productContext?.skuSelector.areAllVariationsSelected
 
+  const productLink = { linkText: product?.linkText, productId: product?.productId }
```

Comment:
You're trying to create these props of this object based on the existence of `product`, but since `product` might be `undefined`, what would happen with the call to `navigate` if both of these props are `undefined`? I believe that you shouldn't call `navigate` if this happens and you shouldn't call `event.preventDefault` and `event.stopPropagation` because you wouldn't use the event